packages/core/test-helpers/core-test-helpers-test-utils/kibana.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "shared-common",
```

Comment:
nit: Is this `shared-server` now? Or, in the future, `test-helpers`?

https://github.com/elastic/kibana/blob/main/dev_docs/operations/packages_idm.mdx#package-types

Reply:
Ah, good catch! I rarely look at the auto-generated files. for now, it's `shared-common` because the package is imported (or will be imported) into more than one type of package.